### DISCUSSION TITLE
Time left for task announcement in Domina Mode

### DISCUSSION THREAD

**Discussion by LunaVoid** [super-productivity/super-productivity#3023] [General] [answered]
Hi I was just wondering if there are other variables that can be used with domina mode. Currently by default it announces the task with ${currentTaskTitle}. is there a way to have it announce the time remaining as well? I've been looking through the code to try and see if I can find the correct stuff but  I must've missed it.

**Comment by LunaVoid** [comment]
Ok found the task.duration but it doesn't seem like there is a currentTaskDuration keyword for that.

**Comment by johannesjo** [comment, marked as answer]
No currently there aren't any other variables. But it should be straightforward to add more. PRs are welcome!

**Comment by LunaVoid** [comment]
Awesome, I'll give it a try when I get a chance!
